templates/posttypes/contributor.blade.php

```diff
@@ -3,82 +3,84 @@
+									<title id="twitter-title-logo">Twitter logo</title>
+									<use href="#twitter-icon"/>
+								</svg>
 							</a>
-						</p>
-					@else
-						<a href="{{$contributor['contributor_twitter']}}" target="_blank">
-							<svg role="img" aria-labelledby="twitter-logo" class="contributor icon-svg">
-								<title id="twitter-title-logo">Twitter logo</title>
-								<use href="#twitter-icon"/>
-							</svg>
-						</a>
+						@endif
 					@endif
-				@endif
-				@if ( $contributor['contributor_linkedin'] )
-					@if ( isset( $exporting ) )
-						<p class="contributor__link">
+					@if ( $contributor['contributor_linkedin'] )
+						@if ( isset( $exporting ) )
+							<p class="contributor__link">
+								<a href="{{$contributor['contributor_linkedin']}}" target="_blank">
+									{{$contributor['contributor_linkedin']}}
+								</a>
+							</p>
+						@else
 							<a href="{{$contributor['contributor_linkedin']}}" target="_blank">
-								{{$contributor['contributor_linkedin']}}
+								<svg role="img" aria-labelledby="linkedin-logo" class="contributor__icon-svg">
+									<title id="linkedin-title-logo">LinkedIn logo</title>
+									<use href="#linkedin-icon"/>
+								</svg>
 							</a>
-						</p>
-					@else
-						<a href="{{$contributor['contributor_linkedin']}}" target="_blank">
-							<svg role="img" aria-labelledby="linkedin-logo" class="contributor icon-svg">
-								<title id="linkedin-title-logo">LinkedIn logo</title>
-								<use href="#linkedin-icon"/>
-							</svg>
-						</a>
+						@endif
 					@endif
-				@endif
-				@if ( $contributor['contributor_github'] )
-					@if ( isset( $exporting ) )
-						<p class="contributor__link">
+					@if ( $contributor['contributor_github'] )
+						@if ( isset( $exporting ) )
+							<p class="contributor__link">
+								<a href="{{$contributor['contributor_github']}}" target="_blank">
+									{{$contributor['contributor_github']}}
+								</a>
+							</p>
+						@else
 							<a href="{{$contributor['contributor_github']}}" target="_blank">
-								{{$contributor['contributor_github']}}
+								<svg role="img" aria-labelledby="github-logo" class="contributor__icon-svg">
+									<title id="github-title-logo">GitHub logo</title>
+									<use href="#github-icon"/>
+								</svg>
 							</a>
-						</p>
-					@else
-						<a href="{{$contributor['contributor_github']}}" target="_blank">
-							<svg role="img" aria-labelledby="github-logo" class="contributor icon-svg">
-								<title id="github-title-logo">GitHub logo</title>
-								<use href="#github-icon"/>
-							</svg>
-						</a>
+						@endif
 					@endif
-				@endif
-			</div>
-		@endif
+				</div>
+			@endif
+		</div>
 	</div>
-	<div class="contributor__bio">{!! wp_kses( $contributor['contributor_description'], true ) !!}</div>
+	<div class="contributor__bio">{!! apply_filters( 'the_content', $contributor['contributor_description'] ) !!}</div>
```

Comment:
Using the same filter used to render front-matter/chapter/back-matter body.

Reply:
In my local this is causing a infinite loop when one of my contributors doesn't have bio I think that filter is not suited for that because is doing like a recursive call maybe just calling our sanitize_string function allowing safe HTML could be a good option I sent a friendly commit

Reply:
Could it be a cache problem?